assets/dev/js/editor/elements/views/base.js

```diff
@@ -621,14 +627,148 @@ BaseElementView = BaseContainer.extend( {
+	 * 'data-binding-setting': 'testimonial_content',      // Setting change to capture, the value will replace the link.
+	 *
+	 * By adding the following example attributes inside the widget the element innerHTML will be linked to the 'testimonial_content' setting value.
+	 *
+	 * Current Limitation:
+	 * Not working with dynamics, will required full re-render.
+	 */
+	linkDataBindings() {
+		/**
+		 * @type {Array.<DataBinding>}
+		 */
+		this.dataBindings = [];
+
+		const id = this.$el.data( 'id' );
+
+		if ( ! id ) {
+			return;
+		}
+
+		const $dataBinding = this.$el.find( '[data-binding-type]' );
+
+		if ( ! $dataBinding.length ) {
+			return;
+		}
+
+		const self = this;
+
+		$dataBinding.filter( function() {
+			const $current = jQuery( this );
+
+			// To support nested data-binding bypass nested data-binding that are not part of the current.
+			if ( $current.closest( '.elementor-element' ).data( 'id' ) === id ) {
+				if ( this.dataset.bindingType ) {
+					self.dataBindings.push( {
+						el: this,
+						dataset: this.dataset,
+					} );
+				}
+			}
+		} );
+	},
+
+	/**
+	 * Function renderDataBindings().
+	 *
+	 * Render linked data.
+	 *
+	 * @param {Object} settings
+	 * @param {Array.<DataBinding>} dataBindings
+	 *
+	 * @returns {boolean} - false on fail.
+	 */
+	renderDataBindings( settings, dataBindings ) {
+		if ( ! this.dataBindings?.length ) {
+			return false;
+		}
+
+		let changed = false;
+
+		for ( const dataBinding of dataBindings ) {
+			switch ( dataBinding.dataset.bindingType ) {
+				case 'repeater-item': {
+					const repeater = this.container.repeaters[ dataBinding.dataset.bindingRepeaterName ];
+
+					if ( ! repeater ) {
+						break;
+					}
+
+					const container = repeater.children.find( ( i ) => i.id === settings.attributes._id );
+
+					if ( ( container?.parent?.children.indexOf( container ) + 1 ) === parseInt( dataBinding.dataset.bindingIndex ) ) {
+						const change = settings.changed[ dataBinding.dataset.bindingSetting ];
+
+						if ( change !== undefined ) {
+							changed = true;
+							dataBinding.el.innerHTML = change;
+						}
+					}
+				}
+				break;
+
+				case 'content': {
+					if ( undefined !== settings.changed[ dataBinding.dataset.bindingSetting ] ) {
+						const change = settings.changed[ dataBinding.dataset.bindingSetting ];
+
+						changed = true;
+						dataBinding.el.innerHTML = change;
+					}
+				}
+				break;
+			}
+
+			if ( changed ) {
+				break;
+			}
+		}
+
+		return changed;
+	},
+
+	/**
+	 * Function renderOnChange().
+	 *
+	 * Render the changes in the settings according to the current situation.
+	 *
+	 * @param {Object} settings
+	 */
+	renderOnChange( settings ) {
+		if ( ! this.allowRender ) {
+			return;
+		}
+
+		// If not a dynamic change, and renderDataBindings() was successful, then it was rendered.
+		if ( ! settings.__dynamic__ && this.renderDataBindings( settings, this.dataBindings ) ) {
```

Comment:
Maybe it's after removing a dynamic tag, so `settings.__dynamic__` is empty, but it needed to re-render